ProcessLib/HydroMechanics/HydroMechanicsFEM-impl.h

```diff
@@ -164,7 +168,18 @@ void HydroMechanicsLocalAssembler<ShapeFunctionDisplacement,
             pressure_size, displacement_size>::Zero(pressure_size,
                                                     displacement_size);
 
+    MaterialLib::Solids::MechanicsBase<DisplacementDim> const& solid_material =
+        *_process_data.solid_materials[0];
+
+    auto linear_elastic_mp =
+        static_cast<MaterialLib::Solids::LinearElasticIsotropic<
```

Comment:
This, of course, is invalid for all other materials.

Reply:
is removed and now uses getBulkModulus